py/makeqstrdata.py

```diff
@@ -227,6 +227,8 @@ def compress(encoding_table, decompressed):
                 current_bit -= 1
     if current_bit != 7:
         current_byte += 1
+    if current_byte > len(decompressed):
+        print("Note: compression increased length", repr(decompressed.decode('utf-8')), len(decompressed), current_byte, file=sys.stderr)
```

Comment:
Does this need to be in a C comment?

Reply:
No, as it goes to the build log (sys.stderr).  I can move it to being in a comment instead if you prefer.

Reply:
Ah, I missed that. This is fine. Thanks!